pages.ja/windows/cmd.md

```diff
@@ -0,0 +1,36 @@
+# cmd
+
+> Windowsコマンドインタープリター
+> 詳しくはこちら: <https://docs.microsoft.com/windows-server/administration/windows-commands/cmd>.
+
+- コマンドインタープリターの新しいインスタンスを開始します:
+
+`cmd`
+
+- 指定されたコマンドを実行して終了します:
+
+`cmd /c "{{command}}"`
```

Comment:
Do you think is clear for non-English speaker if you translate `command`?

Reply:
Yes, I can translate that, Can you check the latest update?